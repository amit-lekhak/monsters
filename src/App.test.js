import { shallow } from 'enzyme';
import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('class component check', () => {
  const wrapper = shallow(<App />);

  expect(wrapper).toMatchSnapshot();
});

it('class component test', () => {
  const wrapper = shallow(<App />);

  expect(wrapper.state('searchField')).toEqual('');

  wrapper.find('SearchBox').simulate('change', {
    target: { value: 'world' },
  });

  // expect(wrapper.state('searchField')).toEqual('world');
  expect(wrapper.find('h1').text()).toEqual('Monsters Rolodex');
});
