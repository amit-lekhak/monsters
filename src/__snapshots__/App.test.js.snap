// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`class component check 1`] = `ShallowWrapper {}`;
